### PR TITLE
hcm: fix smoothing in fall checker

### PR DESCRIPTION
**Pull request by SammyRamone**
## Proposed changes
Fixes the missing smoothing threshold in the FallChecker. It was previously only used when the FallChecker was used through the classifier class.
The threshold prevents invoking fall measurements after a false positive due to message outliers.

## Necessary checks
- [ ] Update package version
- [x] Run `catkin build`
- [x] Write documentation
- [x] Create issues for future work
- [ ] Test on your machine
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

**Comment by SammyRamone** [comment]
still needs some testing before merging to master

File: bitbots_hcm/src/bitbots_hcm/fall_checker.py [hunk 1]
```diff
@@ -54,23 +59,37 @@ def check_falling(self, not_much_smoothed_gyro, quaternion):
...
+                else:
+                    result = self.RIGHT
+
+        if self.smoothing > 0:
+            if result == self.last_result and result != 0:
+                self.counter += 1
+                if self.counter > self.smoothing:
+                    result = result
+                else:
+                    result = 0
```
Comment (Flova) [inline bitbots_hcm/src/bitbots_hcm/fall_checker.py:86]: To clarify the return value we could use `self.STABLE` instad of 0

File: bitbots_hcm/src/bitbots_hcm/fall_checker.py [hunk 2]
```diff
@@ -54,23 +59,37 @@ def check_falling(self, not_much_smoothed_gyro, quaternion):
...
+            else:
+                self.counter = 0
+                result = 0
```
Comment (Flova) [inline bitbots_hcm/src/bitbots_hcm/fall_checker.py:89]: Same here

**Review by Flova — CHANGES_REQUESTED** [review]

**Review by Flova — APPROVED** [review]

**Comment by SammyRamone** [comment]
works like the previous code, there are other issues but I added them here https://github.com/bit-bots/bitbots_motion/issues/200

**Pull request merged**